sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTablesSuite.scala

```diff
@@ -143,6 +163,8 @@ trait ShowTablesSuiteBase extends command.ShowTablesSuiteBase {
  * The class contains tests for the `SHOW TABLES` command to check V1 In-Memory table catalog.
  */
 class ShowTablesSuite extends ShowTablesSuiteBase with CommandSuiteBase {
+  override def version: String = super[ShowTablesSuiteBase].version
```

Comment:
is this line needed?

Reply:
Yes, because both `ShowTablesSuiteBase` and `CommandSuiteBase` override `version`.

Reply:
can we remove this line if we make `ShowTablesSuiteBase` an abstract class?